lms/static/sass/_developer.scss

```diff
@@ -79,4 +79,140 @@
   .global-new, #global-navigation {
     display: none;
   }
+
+  // Copied from _pagination.scss in cms
+  .pagination {
```

Comment:
FYI @frrrances I copied the CSS for the pagination footer from Studio here. It looks a little funny in a few cases a will need cleanup later.
![screen shot 2015-06-25 at 9 08 14 am](https://cloud.githubusercontent.com/assets/6981284/8354941/08290512-1b1a-11e5-96bc-30c45a304047.png)